### PR TITLE
Fix return on success of Weakauras.Import

### LINKED ISSUE #3745: Missing return on succesful import from Weakauras.import in Transmission.lua
### Description

as stated in https://github.com/WeakAuras/WeakAuras2/wiki/API-Documentation I expect the first argument to return a truthy value on succesful importing but it just returns nil. 
Checking the Code  local status, msg = true, "" is set but it is never returned anywhere but it instead returns  ImportNow(data, children, target). That function however does not have a return value.

I might not have explained this properly but hopefully infus understands anyway. Can do a pull request to fix if you want 🤷 

### WeakAuras Version

4.0.2

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Lua Error

_No response_

### Reproduction Steps

1. call Weakauras.Import with a proper table to import
2. import Aura
3. watch the return print and see that it is nil

### Last Good Version

not sure probably before the importing rework

### Screenshots

_No response_

### Export String

_No response_

### PR DESCRIPTION
**Pull request by Jodsderechte**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #3745

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test a
- [ ] Test b

## Checklist

- [ ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

**Comment by Jodsderechte** [comment]
Arguably returning in Weakauras.ImportNow would be closer to the orignial but there is no real gain in that and it brings up the error of it returnil nil when ur in combat eventhough the import is succesfully going to start after combat ends

File: WeakAuras/Transmission.lua [hunk 1]
```diff
@@ -519,7 +518,8 @@ function WeakAuras.Import(inData, target)
   end
 
   tooltipLoading = nil;
-  return ImportNow(data, children, target)
+  ImportNow(data, children, target)
+  return true, ""
```
Comment (mrbuds) [inline WeakAuras/Transmission.lua:522]: Fix should be in `ImportNow`

**Comment by mrbuds** [comment]
As discussed in PM, the documented API for WeakAuras.Import is outdated, the Import process is asynchronous, so if you want the result you need to pass it a callback, that will be executed when the import is successful, fail or is cancel

transmission.lua 488
> local function ImportNow(data, children, target, sender)
> Private.OpenUpdate

weakaurasoptions.lua 886
> function OptionsPrivate.OpenUpdate(data, children, target, sender)
> return frame.update:Open(data, children, target, sender)

optionsframe.lua 550
> frame.update = OptionsPrivate.UpdateFrame(frame)

update.lua 2044
> function OptionsPrivate.UpdateFrame(frame)
>  updateFrame = updateFrame or ConstructUpdateFrame(frame)
>  return updateFrame

update.lua 1245
> Open = function(self, data, children, target, sender)

update.lua 2018
> importButton:SetScript("OnClick", function() group:Import() end);

update.lua 1468
>  Import = function(self)
>    OptionsPrivate.Private.dynFrame:AddAction("import", coroutine.create(function()
>      self:ImportImpl()
>    end))

**Comment by InfusOnWoW** [comment]
Closing until you find the motivation to work on this :)

**Pull request closed without merging**